data/json/items/gunmod/muzzle.json

```diff
@@ -32,10 +32,9 @@
     "color": "dark_gray",
     "location": "muzzle",
     "mod_targets": [ "pistol", "smg", "rifle" ],
-    "damage_modifier": -5,
-    "dispersion_modifier": 15,
-    "loudness_modifier": -30,
-    "recoil_modifier": -15
+    "dispersion_modifier": 120,
```

Comment:
120 dispersion is actually worse than the 5 damage penalty it had before.
Is there no way to impair aiming alone?